specification/consumption/resource-manager/Microsoft.Consumption/stable/2021-05-01/consumption.json

```diff
@@ -5113,6 +5186,16 @@
       "description": "The properties of the credit summary.",
       "type": "object",
       "properties": {
+        "creditCurrency": {
+          "description": "The credit currency.",
+          "type": "string",
+          "readOnly": true
+        },
+        "billingCurrency": {
+          "description": "The billing currency.",
+          "type": "string",
+          "readOnly": true
+        },
         "balanceSummary": {
           "description": "Summary of balances associated with this credit summary.",
           "readOnly": true,
```

Comment:
CreditBalanceSummary definition is missing "EstimatedBalanceInBillingCurrency"

Reply:
Added.